content/applications/sales/sales/send_quotations/different_addresses.rst

```diff
@@ -2,74 +2,54 @@
 Deliver and invoice to different addresses
 ==========================================
 
-With Odoo Sales, you can configure different addresses for delivery and invoicing. For some
-customers, it will be very practical to define specific billing and shipping addresses. Indeed, not
-everyone will have the same delivery location as the invoicing location.
+With Odoo Sales, customers can use different addresses for delivery and invoicing.
 
 Activate the feature
 ====================
 
-Go to :menuselection:`Sales --> Configuration --> Settings` and activate the **Customer Addresses**
-feature.
+Activate the :guilabel:`Customer Addresses` feature in the settings. It is located in
+:menuselection:`Accounting > Configuration > Settings` if the Accounting app is installed, or
+:menuselection:`Invoicing > Configuration > Settings` if the Accounting app is not installed.
```

Comment:
Activate the :guilabel:`Customer Addresses` feature in the settings. 

If the :guilabel:`Accounting` app is installed, the :guilabel:`Customer Addresses` feature is located in :menuselection:`Accounting > Configuration > Settings`.

If the :guilabel:`Accounting` app is *not* installed, the :guilabel:`Customer Addresses` feature is located in
:menuselection:`Invoicing > Configuration > Settings`.